qt/5.x.x/conanfile.py

```diff
@@ -509,8 +507,8 @@ def source(self):
         apply_conandata_patches(self)
         for f in ["renderer", os.path.join("renderer", "core"), os.path.join("renderer", "platform")]:
             replace_in_file(self, os.path.join(self.source_folder, "qt5", "qtwebengine", "src", "3rdparty", "chromium", "third_party", "blink", f, "BUILD.gn"),
-                "  if (enable_precompiled_headers) {\n    if (is_win) {",
-                "  if (enable_precompiled_headers) {\n    if (false) {"
+                "  if (enable_precompiled_headers) {" + os.linesep + "    if (is_win) {",
```

Comment:
Note to self: This doesn't work if git isn't configured as `core.autocrlf=true`. The Windows installer for git supposedly sets that by default, but it still gives the choice so we cannot depend on it. Maybe catching the error and throwing our own one that mentions `core.autocrlf=true` needing to be set would be enough. Or maybe we can set it ourselves somehow when cloning Qt.